pkg/scripts/node.go

```diff
@@ -26,14 +26,21 @@ var (
 	`)
 
 	restartKubeAPIServerCrictlTemplate = heredoc.Doc(`
+		# Disable exit immediately if a command in a pipeline fails.
+		# crictl logs can fail if kubelet fails to set up symlink for the API
+		# server container logs. This usually happens on subsequent script run,
+		# for example when upgrading the cluster.
+		# This is a known issue and it's described here:
+		# https://github.com/kubernetes/kubernetes/issues/52172
+		set +e
```

Comment:
Do we need to restart it back to `set -xeu pipefail` or KubeOne will automatically do that when running the next step?

Reply:
next script invocation will be prefixed with `set -xeu pipefail` automatically.